docs/maui/views/MediaElement.md

```diff
@@ -343,6 +342,28 @@ To read more about handlers, please see the .NET MAUI documentation on [Handlers
 | Stop | Stops playback and resets the position of the current media. |
 | SeekTo | Takes a `TimeSpan` value to set the `Position` property to and takes a `CancellationToken` to cancel the `Task`. |
 
+## Known issues
+
+### Possible exception when using full-screen support on Android
+
+> [!WARNING]
+> Possible exception when using full-screen support on Android.
+
+While this is not an issue with the full-screen support in the .NET MAUI Community Toolkit, using the feature will at present, run the risk of exposing a known issue with .NET MAUI - [MAUI Android build crashes when app is reopened from background. It throws the exception: 'Window was already created.'](https://github.com/dotnet/maui/issues/18692).
+
+#### Solution
+
+To avoid this issue it is recommended to set the `LaunchMode` of the application to `LaunchMode.SingleTask` as per the following example.
+
+```csharp
+[Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, LaunchMode = LaunchMode.SingleTask)]
+public class MainActivity : MauiAppCompatActivity
+{
+}
+```
+
+For a full example of this method included in an application please refer to the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs)
+
```

Comment:
Let's refactor this follow the [`Getting Started` worklflow](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/communication/email?view=net-maui-8.0&tabs=android#get-started) that tells developers to add specific code to specific areas of their app before using `MediaElement`. 

Telling the users to add `LaunchMode.SingeTask` as a required step before using `MediaElement` will allow us to close bug reports that have skipped this step and are experiencing this crash. 

Here's the screenshots from [.NET MAUI's Email Getting Started docs](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/communication/email?view=net-maui-8.0&tabs=windows#get-started). Let's follow this same approach, creating a **Getting Started** section and adding tabs for each supported platform (we'll need to include Tizen too).

| Android Getting Started | iOS / MacCatalyst Getting Started | Windows Getting Started | 
| -- | -- | -- |
| ![](https://github.com/MicrosoftDocs/CommunityToolkit/assets/13558917/5f9739cc-89d7-46eb-a787-35f5648d36e8) | ![](https://github.com/MicrosoftDocs/CommunityToolkit/assets/13558917/fd60678b-eca9-4592-bbce-7c7b9af2a448) | ![](https://github.com/MicrosoftDocs/CommunityToolkit/assets/13558917/2efca5af-95ee-4341-b96f-c4e2f0246daa) |

Reply:
It might be a little tricky to see what it looks like but I have pushed some changes. @jfversluis is there any chance you could grab a screenshot of the Getting started section please?

Reply:
Is it this you're looking for? These are taken from the preview website

![image](https://github.com/MicrosoftDocs/CommunityToolkit/assets/939291/2dadcf6e-f85c-4480-ab2a-70df9c714d79)

![image](https://github.com/MicrosoftDocs/CommunityToolkit/assets/939291/ec29ff02-7722-4032-b74f-ddfef999ab49)

![image](https://github.com/MicrosoftDocs/CommunityToolkit/assets/939291/0b0d02fd-3fae-42b2-84d9-fbc76fbebfe6)

![image](https://github.com/MicrosoftDocs/CommunityToolkit/assets/939291/a8841cfc-1acd-4ca3-a4ee-36960e3d5107)

![image](https://github.com/MicrosoftDocs/CommunityToolkit/assets/939291/65bf4e9f-da92-412b-95ff-d20827a7216f)

Windows looks weird with the extra n/a?